src/components/Board.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Board from './Board'
+
+describe('Board', () => {
+
+  test('shallow mount', () => {
+    const Board = shallow(
+      <Board
```

Comment:
You named the constant here Board which overrode the name that was imported.  

Should be 
```javascript
const board = shallow( <Board updateStatusCallback={()=>{}} />);
```